com.reprezen.swagedit.openapi3/src/com/reprezen/swagedit/openapi3/validation/OpenApi3Validator.java

```diff
@@ -95,7 +95,7 @@ public JsonSchemaValidator getSchemaValidator() {
         // and option in UI is enable.
         if (advancedValidation && nbOfErrors == 0) {
             try {
-                OpenApi3 result = new OpenApi3Parser().parse(baseURI.toURL(), true);
+                OpenApi3 result = new OpenApi3Parser().parse(document.get(), baseURI.toURL(), true);
```

Comment:
Ah, now I know why the string parser was relevant to this issue! I was curious, but didn't give it too much thought.

Reply:
Yes I didn't use string parser because it returned null positions, good to have it fixed.